test/utils/split-props.ts

```diff
@@ -1,5 +1,5 @@
 import test from 'ava'
-import splitProps from '../../src/utils/split-props'
```

Comment:
Looks like only the `tsx` files (not `ts`) are applying spaces surrounding curly braces. Might be nice to fix that so it's internally consistent?

Reply:
Actually this was because my vscode extensions were clashing on formatting .ts and .js files 😅

Reply:
This triggered some more formatting from prettier in the `.ts` files 😬